src/plugins/data/common/es_query/kuery/functions/is.ts

```diff
@@ -46,12 +46,21 @@ export function toElasticsearchQuery(
   const {
     arguments: [fieldNameArg, valueArg, isPhraseArg],
   } = node;
+
+  const isExistsQuery = valueArg.type === 'wildcard' && valueArg.value === wildcard.wildcardSymbol;
+  const isAllFieldsQuery =
+    fieldNameArg.type === 'wildcard' && fieldNameArg.value === wildcard.wildcardSymbol;
```

Comment:
we were using fullFieldNameArg before, is that relevant ?

Reply:
@ppisljar The `fullFieldNameArg` is only used when we are dealing with specific field wildcards, so for example `nested.*` would be matched by that logic. Since this case checks that the fieldName is exactly `*`, we can guarantee that there every field matches.